manifests/1/14/00/14.0.30035.0/14.00.installer.yaml

```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: 14.00
+PackageVersion: 14.0.30035.0
+Installers:
+- Platform:
+  - Windows.Universal
+  MinimumOSVersion: 10.0.10042.0
+  Architecture: x64
+  InstallerType: msix
+  InstallerUrl: https://aka.ms/Microsoft.VCLibs.x64.14.00.Desktop.appx

```

Comment:
```suggestion
  InstallerUrl: https://download.microsoft.com/download/4/7/c/47c6134b-d61f-4024-83bd-b9c9ea951c25/14.0.30035.0-Desktop/Microsoft.VCLibs.x64.14.00.Desktop.appx
```

Reply:
Really? I don’t get any ARP entry when adding it on my VM, and it only shows in Get-AppxPackage after installed